k8s/start.sh

```diff
@@ -0,0 +1,21 @@
+minikube start
+cd ..
+./gradlew build
+eval $(minikube docker-env)
```

Comment:
why do we need it?

Reply:
to rebuild project and use newer version if there were changes